README.md

```diff
@@ -1,95 +1,131 @@
+<div align="center">
+  <a href="https://aztec.network">
+    <img src="https://github.com/AztecProtocol/aztec-packages/blob/master/docs/static/img/aztec-logo.9cde8ae1.svg" alt="Aztec Protocol Logo" width="300">
+  </a>
+</div>
+
 # Aztec Starter
 
 This repo is meant to be a starting point for writing Aztec contracts and tests.
 
-You can find the Easy Private Voting contract in `./src/main.nr`. A simple integration test is in `./src/test/index.test.ts`.
+You can find the **Easy Private Voting contract** in `./src/main.nr`. A simple integration test is in `./src/test/index.test.ts`.
 
 The corresponding tutorial can be found in the [Aztec docs here](https://docs.aztec.network/tutorials/codealong/contract_tutorials/private_voting_contract).
 
-## Getting Started
+<div align="center">
+
+[![GitHub Repo stars](https://img.shields.io/github/stars/AztecProtocol/aztec-packages?logo=github&color=yellow)](https://github.com/AztecProtocol/aztec-packages/stargazers)
```

Comment:
Can you make it more clear that this info is for the aztec-packages repo and not this starter?